src/js/common/schemaform/validation.js

```diff
@@ -343,6 +343,32 @@ export function validateDateRange(errors, dateRange, formData, schema, errorMess
   }
 }
 
+export function validateFile(errors, file) {
```

Comment:
Can we reuse this in validateFileField?

Reply:
The current function being used in the validateFileField map is quite close but slightly different:
```
export function validateFileField(errors, fileList) {
  fileList.forEach((file, index) => {
    let error;
    if (file.errorMessage) {
      error = file.errorMessage;
    } else if (file.uploading) {
      error = 'Uploading file...';
    } else if (!file.confirmationCode) {
      error = 'Something went wrong...';
    }

    if (error && !errors[index]) {
      /* eslint-disable no-param-reassign */
      errors[index] = {
        __errors: [],
        addError(msg) {
          this.__errors.push(msg);
        }
      };
      /* eslint-enable no-param-reassign */
    }

    if (error) {
      errors[index].addError(error);
    }
  });
}

```
In order to reuse this, would we want to write a wrapper function for this which would pass `errors` down, and update this function to accept an index argument, along with some associated conditional logic based on that as well?

Reply:
In that case, maybe we should write a function for just the logic setting the error message.

Reply:
That is much cleaner, thanks, updated.